editor/sidebar/post-format/index.js

```diff
@@ -2,13 +2,13 @@
  * External dependencies
  */
 import { connect } from 'react-redux';
-import { find } from 'lodash';
```

Comment:
I know we're using `flowRight` in Calypso, but I don't see any reason to not just use `flow` instead. I know `compose` aka `flowRight` is more widely used in the community.

Reply:
When using `flowRight` / `compose` you have the order which mirrors how components are wrapped. That might help to make it easier to read:
```js
flowRight(
    connect,
        withAPIData
)(
            Component
);
```